nwb_conversion_tools/utils/ephys_writer.py

```diff
@@ -0,0 +1,57 @@
+from distutils.version import StrictVersion
+
+# import ephys writers
+from .si013nwbephyswriter import SI013NwbEphysWriter
+from .si090nwbephyswriter import SI090NwbEphysWriter
+from .neonwbephyswriter import NEONwbEphysWriter
+
+
+def export_to_nwb(objects_to_write, nwb_file_path, metadata=None, **kwargs):
+    """
+    Writes one or multiple objects to NWB.
+
+    Supported objects are:
+        - spikeinterface version <= 0.13 extractors
+          (spikeextractors.RecordingExtractor / spikeextractors.SortingExtractor)
+        - spikeinterface version >= 0.90 objects
+          (spikeinterface.RecordingExtractor / spikeinterface.SortingExtractor / spikeinterface.WaveformExtractor)
+        - NEO >= 0.10 objects
+          (neo.rawIO / neo.IO)
+
+    Parameters
+    ----------
+    objects_to_write
+    nwb_file_path
+    metadata
+    kwargs
+
+    Returns
+    -------
+
+    """
+    if not isinstance(objects_to_write, list):
+        objects_to_write = [objects_to_write]
+
+    for object_to_write in objects_to_write:
+        writer_class = None
+        try:
+            if isinstance(object_to_write, SI090NwbEphysWriter.supported_types):
+                writer_class = SI090NwbEphysWriter
+        except AssertionError:
```

Comment:
what's going on here?

Reply:
@bendichter we select the correct `writer` class based on the type of the input. The `try` is for uninstalled dependencies